com.zaaane.pianotrainer.yaml

```diff
@@ -0,0 +1,52 @@
+# https://v2.tauri.app/distribute/flatpak/
+# This Flatpak builder configuration was taken from the Tauri docs.
+id: com.zaaane.pianotrainer
+
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+
+command: piano-trainer
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --socket=pulseaudio # Access to PulseAudio, input/output, MIDI
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --env=WEBKIT_DISABLE_COMPOSITING_MODE=1 # Optional: may solve some issues with black webviews on Wayland
+
+modules:
+  - name: binary
+    buildsystem: simple
+
+    sources:
+      - type: file
+        path: com.zaaane.pianotrainer.metainfo.xml
+      - type: file
+        url: https://github.com/ZaneH/piano-trainer/releases/download/app-v1.3.2/Piano.Trainer_1.3.2_amd64.deb
+        sha256: b2ffd1e699655e83417dd233965910e3641c2d5c46494c86438f6a20b29439ca
+        only-arches: [x86_64]
+    build-commands:
+      - set -e
+
+      # Extract the deb package
+      - mkdir deb-extract
+      - ar -x *.deb --output deb-extract
+      - tar -C deb-extract -xf deb-extract/data.tar.gz
+
+      # Copy binary
+      - mkdir -p /app/bin
+      - "install -Dm755 deb-extract/usr/bin/piano-trainer /app/bin/piano-trainer"
+
+      # Copy desktop file + ensure the right icon is set
+      - sed -i 's/^Icon=.*/Icon=com.zaaane.pianotrainer/' deb-extract/usr/share/applications/Piano\ Trainer.desktop
```

Comment:
NEVER use sed for changing the desktop file. there is the `desktop-file-edit` command